Makefile

```diff
@@ -83,29 +83,29 @@ dbdocs:
 	$(DOCKER_CMD) dot -Tpdf doc/database/ledgersmb.dot -o doc/database/ledgersmb.pdf
 
 npm_install:
-	$(DOCKER_CMD) npm $(NPM_COMMAND) --no-save
+	$(DOCKER_CMD) cd UI && npm $(NPM_COMMAND) --no-save
 
 dojo: npm_install
-	$(DOCKER_CMD) npm run build
+	$(DOCKER_CMD) cd UI && npm run build
 
 devdojo: npm_install
-	$(DOCKER_CMD) npm run build:dev
+	$(DOCKER_CMD) cd UI && npm run build:dev
 
 js: npm_install
-	$(DOCKER_CMD) npm run build
+	$(DOCKER_CMD) cd UI && npm run build
 
 jsdev: npm_install
-	$(DOCKER_CMD) npm run build:dev
+	$(DOCKER_CMD) cd UI && npm run build:dev
 
 jslint:
 ifneq ($(origin FIX),undefined)
-	$(DOCKER_CMD) npm run lint:js:fix
+	$(DOCKER_CMD) cd UI && npm run lint:js:fix
 else
-	$(DOCKER_CMD) npm run lint:js
+	$(DOCKER_CMD) cd UI && npm run lint:js
```

Comment:
Doesn't this cause the "npm" command to be run outside of the docker environment?

Reply:
No